contract/tests/tests.rs

```diff
@@ -37,3 +38,19 @@ async fn test_contract_can_not_be_reinitialized() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn test_old_state_can_be_migrated_to_v0() -> anyhow::Result<()> {
+    let old_contract = MpcContract::init(3, BTreeMap::new());
+    env::state_write(&old_contract);
```

Comment:
This code is not executed in Wasmer. Where state is written to?
We need to write an integration test using workspaces to test this logic. And to test it properly we will need to have actual old and new states.

Reply:
We can have a dummy structure representing an old state V0, then V1 will be the current one. I don't think that is necessary now. We will add such a test on the next state change.

Reply:
The `env::state_write` writes to a `MockBlockchain` in the case of a unit test. We don't need to go into workspaces for this since all this is testing is the serialization and that's what we want to test directly. 

> We can have a dummy structure representing an old state V0, then V1 will be the current one. I don't think that is necessary now. We will add such a test on the next state change.

yeah, we need to maintain a list of all structs that we used before in these tests